csharp/ql/src/experimental/Security Features/campaign/Solorigate/ModifiedFnvFunctionDetection.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @name Detects a modified FNV function
+ * @description Possible indication of Solorigate. Detects a modified FNV1 function, where there is an additional xor using a literal after the regular FNV hash
+ * @kind problem
+ * @tags security
+ *       solorigate
+ * @precision high
+ * @id cs/solorigate/modified-fnv-function-detection
+ * @problem.severity error
+ */
+
+import csharp
+import Solorigate
+import microsoft.code.csharp.Cryptography.NonCryptographicHashes
+
+from Variable v, Literal l, LoopStmt loop, Expr additional_xor
```

Comment:
The qhelp states that `l` has the value `6605813339339102567`, but that is not checked in the query.

Reply:
Thanks, we removed the literal and forgot to change the text. Fixing it